src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      state.books.push(
+        {
+          ...payload,
+          author: 'Suzanne Collins',
+          progress: 0,
+          state: 'Chapter 1',
+        },
+      );
+    },
+    removeBook: (state, { payload }) => {
+      /* eslint-disable no-param-reassign */
+      state.books = state.books.filter((book) => (book.title !== payload.title
+        && book.category !== payload.category));
```

Comment:
**OPTIONAL**- Since all books have their unique `IDs`, it will be more efficient to delete books using their `IDs` instead, please fix this 🙏